easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.1.0-fosscuda-2020b.eb

```diff
@@ -0,0 +1,189 @@
+            'e02e1298b59038078a2107e1c04b14172898e1f79b7f3afc2394047db532e228',  # AlphaFold-2.1.0_data-dep-paths.patch
+            'dfda4dd5f9aba19fe2b6eb9a0ec583d12dcefdfee8ab8803fc57ad48d582db04',  # AlphaFold-2.0.0_n-cpu.patch
+            '5363d403baf5ab73f4d3ddd72e19af9ff832de4b1d7ba25a5fbcc5846c1c890f',  # AlphaFold-2.1.0_fix-scp-path.patch
+            '1a2e4e843bd9a4d15ee39e6c37cc63ba281311cc7a0a5610f0e43b52ef93faac',  # AlphaFold-2.0.1_setup_rm_tfcpu.patch
+            '3b9ee75aba872d912e2bca401a794054d652d47fb9e87e53e99781cef3c3091f',  # AlphaFold-2.0.0_relax_gpu.patch
+        ],
+        'start_dir': 'alphafold-%(version)s',
+        'use_pip': True,
+    }),
+]
+
+use_pip = True
+
+exts_list = [
+    ('PDBFixer', '1.7', {
+        'source_urls': ['https://github.com/openmm/pdbfixer/archive/refs/tags/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['a0bef3c52a7bbe69a6aea5333f51f3e7d158339be5829aed19b0344bd66d4eea'],
+    }),
+    ('toolz', '0.11.1', {
+        'checksums': ['c7a47921f07822fe534fb1c01c9931ab335a4390c782bd28c6bcc7c2f71f3fbf'],
+    }),
+    ('chex', '0.0.8', {
+        'checksums': ['d6ce1329470116b6f172a72b1131bfd4d11fb7eb465e6077c3b36224b5b09fe4'],
+    }),
+    ('tabulate', '0.8.9', {
+        'checksums': ['eb1d13f25760052e8931f2ef80aaf6045a6cceb47514db8beab24cded16f13a7'],
+    }),
+    ('dm-haiku', '0.0.4', {
+        'modulename': 'haiku',
+        'source_urls': ['https://github.com/deepmind/dm-haiku/archive/refs/tags/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': '%(name)s-%(version)s.tar.gz'}],
+        'checksums': ['e9896d161938b53f869fde207f3f6fca496d09b2a47d21dd9b6b65e897ad6aab'],
+    }),
+    ('dm-tree', '0.1.6', {
+        'modulename': 'tree',
+        'checksums': ['6776404b23b4522c01012ffb314632aba092c9541577004ab153321e87da439a'],
+    }),
+    ('websocket-client', '1.2.1', {
+        'modulename': 'websocket',
+        'checksums': ['8dfb715d8a992f5712fff8c843adae94e22b22a99b2c5e6b0ec4a1a981cc4e0d'],
+    }),
+    ('docker', '5.0.0', {
+        'checksums': ['3e8bc47534e0ca9331d72c32f2881bb13b93ded0bcdeab3c833fb7cf61c0a9a5'],
+    }),
+    ('immutabledict', '2.1.0', {
+        'checksums': ['673fb8f30f46d23dd394050b979f5b7f4c5398982b99ebc854fb873e646b967a'],
+    }),
+    ('contextlib2', '21.6.0', {
+        'checksums': ['ab1e2bfe1d01d968e1b7e8d9023bc51ef3509bba217bb730cee3827e1ee82869'],
+    }),
+    ('ml_collections', '0.1.0', {
+        'preinstallopts': "touch requirements.txt && touch requirements-test.txt && ",
+        'checksums': ['59a17fcd1c140153009788517f304caaddd7a94f06690f9f0ed09987beebcf3c'],
+    }),
+]
+
+postinstallcmds = [
+    "mkdir -p %(installdir)s/bin",
+    # run_alphafold.py script is missing a shebang...
+    "echo '#!/usr/bin/env python' > %(installdir)s/bin/run_alphafold.py",
```

Comment:
Is this something `fix_python_shebang_for` should be doing (from https://docs.easybuild.io/en/latest/version-specific/easyconfig_parameters.html), or it the line simply missing and the shebang fixer doesn't work in that case?

Reply:
@terjekv I did not try to use fix_python_shebang_for. The postinstallcmds were inherited from the previous versions of this EC

Reply:
@SebastianAchilles @terjekv It turned out that the AlphaFold-2.1.0_data-dep-paths.patch is missing some lines to treat the case --model_preset=multimer. Furthermore there is a typo in line 34 (should be uniprot.fasta and not uniprot.data). I'll try to fix this tomorrow.

Reply:
I don't think `fix_python_shebang_for` will work here, because there's no shebang line at all...

We could enhance it to also cover that case, but it doesn't currently, it expects a `^#!` line on top...